lib/elixir/src/elixir_with.erl

```diff
@@ -1,7 +1,7 @@
 -module(elixir_with).
 -export([expand/3, translate/3]).
 -include("elixir.hrl").
-
+-define(ref(Meta), {ref, Meta, elixir_with}).
```

Comment:
Can we define it directly in code? I think it brings more indirection.
`elixir_with` –> just `nil`.